dropshot/src/handler.rs

```diff
@@ -202,7 +203,7 @@ pub trait Extractor: Send + Sync + Sized {
  * the associated endpoint is paginated.
  */
 pub struct ExtractorMetadata {
-    pub paginated: bool,
+    pub extension_mode: ExtensionMode,
```

Comment:
@lifning it looks like this was a breaking change:

```
error[E0560]: struct `dropshot::ExtractorMetadata` has no field named `paginated`
  --> nexus/src/authn/external/cookies.rs:51:29
   |
51 |         ExtractorMetadata { paginated: false, parameters: vec![] }
   |                             ^^^^^^^^^ `dropshot::ExtractorMetadata` does not have this field
   |
   = note: available fields are: `extension_mode`, `parameters`
```

https://github.com/oxidecomputer/omicron/pull/1626

Could you please update the changelog?

Reply:
https://github.com/oxidecomputer/dropshot/pull/428